### DISCUSSION TITLE
Support for Tolino Vision 5 (Kobo Libra H2O clone)

### DISCUSSION THREAD

**Discussion by shyney7** [koreader/koreader#7167] [Q&A] [answered]
Hi,

is it possible to install Koreader on the Tolino Vision 5? It's a Kobo Libra H2O clone but with another firmware I think. But still based on Android.

**Comment by Frenzie** [comment]
Probably, but you'll have to use the Android version.

**Comment by Galunid** [comment]
That shouldn't be a problem, but you'll have to use android version (not kobo). It won't run as well as it'd run on libra though.
@zwim can probably tell you a bit more?

**Comment by ghost** [comment, marked as answer]
You can install the Android version of KOReader on the Tolino Vision 5 (not tested, but I have it on the Epos2 and the Vision4Hd).

1. First you need to activate "adb-root". (see: https://allesebook.de/anleitung/tolino-vision-5-adb-root-980665/ )
With that you can install and run KOReader. (For changing the warmth of the frontlight and for the frontlight button you will use the original Tolino-app.)
2. Nice to have is a launcher (see "Simple Ink Launcher" https://www.mobileread.com/forums/showthread.php?t=320940 or "Smart Launcher"
3. I have installed the app "Button Mapper", so that I can switch to the android launcher with a tripple-click on the page-up-button.
4. If you want to change the  frontlight warmth from within KOReader (which is a really cool feature) you would need a supersu.
-> Find the correct TWRP for the Vision 5 (ask on https://forum.alles-ebook.de/) and install it.
-> get SuperSu Free (I have v2.82 running).
-> install it from the recovery.

I found all the necessary infos on https://forum.alles-ebook.de/. If you have problems, please ask in that forum.
